modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -179,28 +174,25 @@ After you create a streaming channel, you can start receiving events by subscrib
 </flow>
 ----
 
-A Mule flow is divided in two. The first portion is usually an inbound endpoint (or an HTTP connector) and a message source. The Mule flow is an entity that receives and generates events that later are processed by the rest of the flow. The other portion is a collection of message processors that processes the messages (also known as events) that are received and generated by the inbound endpoint.
-
-Every time a subscription to `/u/TestStreaming` receives an event, it executes the rest of the flow. In the case of this example, it logs a message at INFO level.
-
-NOTE: The *Streaming Channel* field of the Subscribe Channel Listener operation does not display change events that are available in the Salesforce environment because this information is not provided in the Salesforce API. However, your connector can subscribe to a streaming channel. For example, to subscribe to the `All Change Events` channel, use `/data/ChangeEvents` as the channel name to which you subscribe. See https://developer.salesforce.com/docs/atlas.en-us.change_data_capture.meta/change_data_capture/cdc_subscribe_channels.htm[Subscription Channels] in the Salesforce Change Data Capture Developer Guide.
+The *Streaming Channel* field of the Subscribe Channel Listener operation does not display change events that are available in the Salesforce environment, because this information is not provided in the Salesforce API. However, your connector can subscribe to a streaming channel to obtain this information. For example, to subscribe to the `All Change Events` channel, use `/data/ChangeEvents` as the channel name to which you subscribe.
 
+For more information, see https://developer.salesforce.com/docs/atlas.en-us.change_data_capture.meta/change_data_capture/cdc_subscribe_channels.htm[Subscription Channels] in the Salesforce Change Data Capture Developer Guide.
 
 === Streaming Channel Inbound Properties
 
 This information gets passed along as inbound properties:
 
-* `channel` - Maps to the Channel JSON property.
-* `type` - Maps to the Type JSON property in data.
-* `createdDate` - Maps to the createdDate JSON property in data.
+* `channel` maps to the Channel JSON property.
+* `type` maps to the Type JSON property in data.
+* `createdDate` maps to the createdDate JSON property in data.
 
 Except for `channel`, each property inside an event is available as an inbound property.
 
-=== Replay Events from a Streaming Channel
+// This does not make sense.
```

Comment:
What do you mean? It makes sense you can either Subscribe to a channel or Replay the events of a channel